elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAsyncConfiguration.java

```diff
@@ -49,7 +62,9 @@
 @EntityScan(basePackageClasses = AsyncQuery.class)
 @EnableConfigurationProperties(ElideConfigProperties.class)
 @ConditionalOnExpression("${elide.async.enabled:false}")
-public class ElideAsyncConfiguration {
+@EnableAsync
```

Comment:
Why did we add these two annotations?

Reply:
This is for providing executors and controlling other configs so that controller can run the export in the background instead of blocking the threads. I think we can move it out of auto-configure and provide as a reference in the example repo.